src/Navigation.jsx

```diff
@@ -154,7 +155,7 @@ const Navigation = () => {
           <Route path="user" element={<UserPage />} />
         </Route>
 
-        <Route path="reports">
+        <Route path="reports" element={<ReportsLayout />}>
```

Comment:
Should we do the same thing for settings?

Reply:
It seems to me that yes, in the components that have the same design.
It would be necessary to analyze it because I do not see that the state update generated by the socket is rendering the settings menu.

**I apologize for any translation errors, my language is Spanish and I am using a translator.**

Reply:
Can you please update settings as well, so we have consistent codebase.

Reply:
Of course, let me see what I can do.